src/cljs/rems/extra_pages.cljs

```diff
@@ -0,0 +1,49 @@
+(ns rems.extra-pages
+  (:require [markdown.core :as md]
+            [re-frame.core :as rf]
+            [rems.text :refer [text]]
+            [rems.util :refer [fetch]]
+            [rems.spinner :as spinner]))
+
+;;;; State
+
+(rf/reg-event-fx
+ ::enter-page
+ (fn [{:keys [db]} [_ page-id]]
+   {:db (assoc db ::loading? true)
+    ::fetch-extra-page [page-id]}))
+
+(defn- fetch-extra-page [page-id]
+  (fetch (str "/api/extra-pages/" page-id)
+         {:handler #(rf/dispatch [::fetch-extra-page-result %])}))
+
+(rf/reg-fx ::fetch-extra-page (fn [[page-id]] (fetch-extra-page page-id)))
```

Comment:
you can inline fetch-extra-page here, see docs/architecture/004-reframe-style.md

Reply:
Done, thanks for pointing to the document, I removed the extra layer of indirection as per the document's advice